### ISSUE TITLE
Screen reader related accessibility issues

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#643]
A few minutes of testing with a screen reader revealed several problems in the Tobira user interface. Note that I did not do a systematic test, I basically just visited Tobira trying out a screen reader setup. I'm not separating these into several issues, even though they are unrelated problems and you can fix them separately:

- [ ] Description of “Home” link
- [ ] Page navigation links with no description
- [ ] Header link functionality description missing

---

##### Description of “Home” link

The first thing you access on the page is the “Home” link (the logo). But the text I get from a screen reader is something like “Link – Logo of Test University”. I don't care what the image depicts and it isn't really relevant in this context. What is relevant is, that the functionality of this is “Go to home page”. Unfortunately, that is not described.

![Screenshot from 2022-12-29 12-35-35](https://user-images.githubusercontent.com/1008395/209946979-df44c186-9e84-442e-b90f-8f85f91cf50a.png)

---

##### Page navigation links with no description

In “My Videos”, tabbing to the page navigation links, there is no description what these links do. I'm just informed that these are links. That makes them impossible to use if I cannot see the icons.

Additionally, I can still access seemingly deactivated links. See #642 for more details.

![Screenshot from 2022-12-29 12-41-12](https://user-images.githubusercontent.com/1008395/209947539-60f09db7-fd4f-4900-92c5-3a5446b06620.png)

---

##### Header link functionality description missing

In “My Videos”, tabbing to the headers of the video table, the screen reader informs me that I'm e.g. on the “Created” link. But the actual functionality of these links is sorting. That is not explained. More than that, I also don't know the current sort order and what would happen if I trigger the link.

![Screenshot from 2022-12-29 12-36-25](https://user-images.githubusercontent.com/1008395/209947837-aadfd08e-941d-47be-ab88-2dddc8146698.png)

---

We should do a proper screen reader test next year.

**Comment by naweidner** [comment]
I also did a screen reader test and found two main issues (buttons and links). I mixed these issues with some general/additional informations. 

Also I'm not sure if I still need to adjust my screen readers settings (Orca and NVDA) or if I'm doing something wrong. My screen readers don't read tooltips, so I don't get the additional informations the tooltips provide.

--- 

**Buttons:** 
Links or other elements that behave like buttons (custom components) should at least have the attribute role="button" so that a screen reader can recognize it as a button and can pass this information on to the user. 
If a button only has an icon you can add aria-label to make them accessible for users with screen readers. 

**Examples/Issues:** 
The three dots menu ("User settings") next to the login button does not have an accessible name.  
Same for the three icons in small views. The screen reader just says "Schalter" ("Button"). 

![GetImage(4)](https://user-images.githubusercontent.com/93541078/211803346-52f0b945-e802-42b0-8abb-b256235befac.png)
![GetImage(5)](https://user-images.githubusercontent.com/93541078/211803364-94a7078e-ef07-4b99-ba24-00a78af39cfd.png)

When "user settings" is expanded, I would expect the screen reader to tell me whether the language that currently has focus is selected or not. In this case I would expect the screen reader to say something like "Englisch selected" (or "Englisch not selected" when the currently selected language is German). 

![GetImage(6)](https://user-images.githubusercontent.com/93541078/211803645-e59f5d5c-b11d-401f-940e-8b3997951e95.png)

---

**Links:** 
Use the `<a/>` tag for links if possible, otherwise add role="link" to the element. 
A link should have a descriptive link-text and should make sense without the context (screen reader users can jump from link to link and the screen reader will only read the link-text), if not possible add aria-label. When using aria-label screen readers ignore the link-text. 

**Example/Issues:**  
"Opencast" and "here" are not descriptive link-texts, the user will not get any information about what happens, when clicking on these links. The user will only hear "Opencast link" and "here link". 

I would suggest to add an aria-label in this case. 
Same goes for "GraphiQL" in the footer. 

![GetImage(7)](https://user-images.githubusercontent.com/93541078/211804118-7a5aca4c-b02e-45e4-8c98-4a98455a0108.png)

On this home icon the screen reader gives me different output: 
- NVDA - Firefox 108: reads the target url 
- NVDA - Chrome 108: "home icon link" 
- Orca - Firefox 107: reads only "link" 

![GetImage(8)](https://user-images.githubusercontent.com/93541078/211804132-9343541a-642a-4c37-896b-798a46f52f14.png)
